src/ImageReconstruction.jl

```diff
@@ -74,29 +73,42 @@ function iradon(sinogram::AbstractMatrix, θ::AbstractRange, t::AbstractRange)
     K = length(θ)
     Npad = nextpow(2, 2 * N - 1)
     τ = step(t)
-    ramp = fft(_zero_pad(_ramp_spatial(N, τ), Npad - N))
+    ramp = fft(_ramp_spatial(N, τ, Npad))
     i = div(N, 2) + 1
     j = i + N - 1
 
-    image = zeros(eltype(sinogram), pixels, pixels)
-    Q = Vector{eltype(sinogram)}(undef, N)
+	T = eltype(sinogram)
+	image = [zeros(T, pixels, pixels) for _ = 1:nthreads()]
+	P′ = [Vector{Complex{T}}(undef, Npad) for _ in 1:nthreads()]
+	Q = [Vector{T}(undef, N) for _ in 1:nthreads()]
+
+	l = SpinLock()
+	@inbounds @threads for (k, θₖ) in collect(enumerate(θ))
```

Comment:
I think it makes more sense to put `@inbounds` in the inner loop

Note the mixed usage of TABs and SPACES

Reply:
That TABS and SPACES surprise me. I use the Format Code feature in Atom. I’ll have to be more careful. 

Can you elaborate on restricting @inbounds to the inner loop?

Reply:
> Can you elaborate on restricting `@inbounds` to the inner loop?

🤔 This is not very informative. My understanding here is that there won't be much performance gain by disabling bounds checking for the outer loop, since `length(theta)` is relatively small compared to `pixels*pixels`. That's why I think it's more reasonable to put `@inbounds` to the inner loop, which looks more like a hot spot to me.

I didn't do any benchmark here, so it's very likely I said it wrong and that putting `@inbounds` in the outer loop does improve the performance. (Perhaps you've already known that)

Reply:
I did a quick comparison with BenchmarkTools and found no significant difference between either @inbound location.